arkane/common.py

```diff
@@ -590,3 +591,94 @@ def get_element_mass(input_element, isotope=None):
+    Either ``numbers`` or ``symbols`` must be given.
+
+    Args:
+        coords (np.array): Entries are 3-length lists of xyz coordinates for an atom.
+        numbers (np.array, list): Entries are atomic numbers corresponding to coords.
+        symbols (list): Entries are atom symbols corresponding to coords.
+
+    Returns:
+        np.array: The center of mass coordinates.
+    """
+    if symbols is None and numbers is None:
+        raise IndexError('Either symbols or numbers must be given.')
+    if numbers is not None:
+        symbols = [symbol_by_number[number] for number in numbers]
+    center, total_mass = np.zeros(3, np.float64), 0
+    for coord, symbol in zip(coords, symbols):
+        mass = get_element_mass(symbol)[0]
+        center += mass * coord
+        total_mass += mass
+    center /= total_mass
+    return center
+
+
+def get_moment_of_inertia_tensor(coords, numbers=None, symbols=None):
+    """
+    Calculate and return the moment of inertia tensor for the current
+    geometry in amu*angstrom^2. If the coordinates are not at the center of mass,
+    they are temporarily shifted there for the purposes of this calculation.
+    Adapted from J.W. Allen: https://github.com/jwallen/ChemPy/blob/master/chempy/geometry.py
+
+    Args:
+        coords (np.array): Entries are 3-length lists of xyz coordinates for an atom.
+        numbers (np.array, list): Entries are atomic numbers corresponding to coords.
+        symbols (list): Entries are atom symbols corresponding to coords.
+
+    Returns:
+        np.array: The 3x3 moment of inertia tensor.
+    Raises:
+        InputError: If neither ``symbols`` nor ``numbers`` are given, or if they have a different length than ``coords``
+    """
+    if symbols is None and numbers is None:
+        raise InputError('Either symbols or numbers must be given.')
+    if numbers is not None:
+        symbols = [symbol_by_number[number] for number in numbers]
+    if len(coords) != len(symbols):
+        raise InputError(f'The number of atoms ({len(symbols)}) is not equal to the number of '
+                         f'atomic coordinates ({len(list(coords))})')
+    tensor = np.zeros((3, 3), np.float64)
+    center_of_mass = get_center_of_mass(coords=coords, numbers=numbers, symbols=symbols)
+    for symbol, coord in zip(symbols, coords):
+        mass = get_element_mass(symbol)[0]
+        cm_coord = coord - center_of_mass
+        tensor[0, 0] += mass * (cm_coord[1] * cm_coord[1] + cm_coord[2] * cm_coord[2])
+        tensor[1, 1] += mass * (cm_coord[0] * cm_coord[0] + cm_coord[2] * cm_coord[2])
+        tensor[2, 2] += mass * (cm_coord[0] * cm_coord[0] + cm_coord[1] * cm_coord[1])
+        tensor[0, 1] -= mass * cm_coord[0] * cm_coord[1]
+        tensor[0, 2] -= mass * cm_coord[0] * cm_coord[2]
+        tensor[1, 2] -= mass * cm_coord[1] * cm_coord[2]
+    tensor[1, 0] = tensor[0, 1]
+    tensor[2, 0] = tensor[0, 2]
+    tensor[2, 1] = tensor[1, 2]
+    return tensor
```

Comment:
I could n't recreate the Gaussian output for CH4 manually. I used excel to do the maths.